Gems/Atom/Feature/Common/Assets/Shaders/Shadow/FullscreenShadow.shadervariantlist

```diff
@@ -0,0 +1,23 @@
+{
+    "Shader" : "FullscreenShadow.shader",
+    "Variants": [
+        {
+            "StableId": 1,
+            "Options": {
+                "o_msaaMode": "MsaaMode::MsaaNone"
+            }
+        },
+        {
+            "StableId": 2,
+            "Options": {
+                "o_msaaMode": "MsaaMode::Msaa2x"
+            }
+        },
+        {
+            "StableId": 3,
```

Comment:
Mentioned in the PR description - I left them out because they're rarely used so I don't want to force everyone to pay the cost of compiling those variants. It'll just use the root variant if it can't find a baked one.